native_client/ctcdecode/Makefile

```diff
@@ -23,15 +23,22 @@ clean-keep-common:
 
 clean: clean-keep-common
 	rm -f common.a
+	rm workspace_status.cc
+	rm -fr bazel-out/
 
-bindings: clean-keep-common
+workspace_status.cc: ../workspace_status.h
```

Comment:
This file depends on all the different sources of version info, not on the header. The header almost never changes. I'm pretty sure this rule will lead to stale versions if you rebuild without doing a `make clean`.

Reply:
We could enforce it, or make the target `PHONY` ?